x/concentrated-liquidity/genesis_test.go

```diff
@@ -82,12 +85,15 @@ func setupGenesis(baseGenesis genesis.GenesisState, poolGenesisEntries []singleP
 		if err != nil {
 			panic(err)
 		}
-		baseGenesis.PoolData = append(baseGenesis.PoolData, &genesis.PoolData{
-			Pool:  poolAny,
-			Ticks: poolGenesisEntry.tick,
+		baseGenesis.PoolData = append(baseGenesis.PoolData, genesis.PoolData{
```

Comment:
Please update godoc that now position id, incentive records and accumulators are also added

Reply:
Godoc updated :)